arm-sql/2014-04-01/swagger/replicationLinks.json

```diff
@@ -0,0 +1,394 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure SQL Database API spec",
+    "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases.",
```

Comment:
Good point. While generating a Composite client, the title and description of the composite swagger spec matters. However it would be nice if we can keep the title and description of individual swagger specs, relevant to the stuff present in that spec. Hence the title and description should refer to replicationlinks over here.